Makefile

```diff
@@ -1,17 +1,18 @@
-#Only use if using Xcode 12+
-PREFIX=$(THEOS)/toolchain/Xcode.xctoolchain/usr/bin/
-
-export ARCHS = arm64 arm64e
-TARGET := iphone:clang:13.4
+ARCHS = arm64 arm64e
```

Comment:
In my opinion, I see little benefits of having arm64e for App Stores tweaks. It's true that arm64e will be preferred over arm64 when injected as a dylib into the executable - but for most of the cases we just want to support 64-bit: just arm64.